.zenodo.json

```diff
@@ -0,0 +1,83 @@
+{
+    "title": "Plots.jl",
+    "license": "MIT", 
+    "creators": [
+        {
+            "affiliation": "Elemental Cognition", 
+            "name": "Tom Breloff"
+        }, 
+        {
+            "affiliation": "TU Wien", 
+            "name": "Daniel Schwabeneder"
+            "orcid": "0000-0002-0412-0777"
+        }, 
+        {
+            "affiliation": "GLOBE Institute", 
+            "name": "Michael Krabbe Borregaard"
+        }, 
+        {
+            "affiliation": "Leibniz Universit\u00e4t Hannover", 
+            "name": "Simon Christ",
+            "orcid": "0000-0002-5866-1472"
+        }, 
+        {
+            "affiliation": "Forschungszentrum J\u00fclich", 
+            "name": "Josef Heinen"
+        }, 
+        {
+            "name": "Yuval"
+        }, 
+        {
+            "name": "Andrew Palugniok"
+        }, 
+        {
+            "affiliation": "@beacon-biosignals", 
+            "name": "Simon"
+        }, 
+        {
+            "affiliation": "Veos Digital (https://veos.digital/)", 
+            "name": "Pietro Vertechi"
+        }, 
+        {
+            "name": "Zhanibek"
```

Comment:
name: Zhanibek Omarov
orcid: https://orcid.org/0000-0002-8783-8791

aff: Korea Advanced Inst. of Science and Technology (KAIST)